tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { of } from 'rxjs';
+import { DataService } from '../../src/app/core/services/data.service';
+import { GithubService } from '../../src/app/core/services/github.service';
+import { autoSpy } from '../auto-spy';
+import { csvString, dataFile, dataFileTeamStructure } from '../constants/data.constants';
+
+describe('DataService', () => {
+  it('when getDataFile is called it should return the correct data', done => {
+    // arrange
+    const { build, githubService } = setup().default();
+    const c = build();

```

Comment:
Should we use a more descriptive name rather than `c`?
Perhaps `dataService` instead of `c` ?